specification/dimensions/chargefrequency.md

```diff
@@ -0,0 +1,43 @@
+# Charge Frequency
+
+Charge Frequency indicates how often a charge would occur - periodically, usage-based, or one-time charge. When the charge frequency is periodic (typically monthly), it can be inferred that this type of charge can be seen in past and future bill periods (within constraints like charge type or possible expiry). Such records indicate charges for the acquisition of a service or resource bought upfront or on a recurring basis.
```

Comment:
A few thoughts:
1. I believe we try to avoid listing the values in the description.
2. The "inferred" comment feels odd here. The intent is spot on, but this may be the first or last charge. Curious what others think.
3. "Billing periods" (not "bill periods").
4. Should we link the Charge Frequency to Charge Period Start/End? How do those relate? Can we use that to know when the charge will recur?

Reply:
1. I have turned it into a few example values
2. Removed inferred
3. changed 
4. I don't have a strong opinion either way. I do see the benefit of having a start and end to understand when the charge will recur or until when, but I know we want to be really discriminate about adding more dimensions

Reply:
ChargePeriodStart/End already exist :-) I was referring to the ones we added in v0.5.

Reply:
In this case, the charge period start and end refer to the month-year that the recurring charge has started and when that charge will expire or stop showing up, correct?